pom.xml

```diff
@@ -240,6 +231,96 @@
         </plugins>
     </build>
     <profiles>
+        <profile>
+            <id>jdk11</id>
+            <activation>
+                <jdk>11</jdk>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-compiler-plugin</artifactId>
+                        <version>3.8.0</version>
+                        <configuration>
+                            <release>11</release>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+            <dependencies>
+                <dependency>
+                    <groupId>org.glassfish.jaxb</groupId>
+                    <artifactId>jaxb-runtime</artifactId>
+                    <version>2.4.0-b180830.0438</version>
+                </dependency>
+                <dependency>
+                    <groupId>com.sun.activation</groupId>
+                    <artifactId>javax.activation</artifactId>
+                    <version>1.2.0</version>
+                </dependency>
+            </dependencies>
+        </profile>
+        <profile>
+            <id>jdk10</id>
+            <activation>
+                <jdk>10</jdk>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-compiler-plugin</artifactId>
+                        <version>3.8.0</version>
+                        <configuration>
+                            <release>10</release>
+                            <compilerArgs>
+                                <arg>--add-modules=java.xml.ws.annotation</arg>
```

Comment:
Is there a reason, why you only add this to JDK10 and not to JDK11?

Reply:
It doesn't work for JDK11. JDK11 uses the dependencies instead (see last 2 commits).

Reply:
Is it different from including java.xml.bind and javax.activation modules dependencies?

Reply:
An additional question:
Which of our maven dependencies requires these modules?
I mean as far as I know these modules are part of some Java EE APIs, which have been removed in JDK11 from Java SE.
This indicates, that we have some maven dependencies, which require Java EE APIs.
Are these dependencies Java 11 compatible?

Reply:
@qparis the functionality should be the same. However, the modules are available in Java EE until JDK 10 (though deprecated) and have been removed in JDK 11. So technically, the maven dependencies are probably a bit different from the modules in Java 10.

@madoar as far as I understand, the maven dependencies are made to provide the removed functionality of the Java EE APIs for Java versions where those have been removed. So for Java 11, we do not use the Java EE APIs anymore but instead rely on those maven dependencies.